Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ruby:2.4-alpine
+
+RUN apk --update add grep
+
+WORKDIR /usr/src/app
+COPY . /usr/src/app
+
+RUN gem install -g
```

Comment:
This is a way to install Gemfile without bundler. Rubygems support this option for a while now.